src/karmen_frontend/cypress/integration/organizations/editing_spec.js

```diff
@@ -17,29 +17,37 @@ describe("Organizations: Editing", function () {
   });
 
   it("fails with no name", function () {
-    cy.get("input#name").clear();
-    cy.get('button[type="submit"]').click();
-    cy.get("form").contains("Name cannot be empty");
+    cy.get("input#username").type(email);
+    cy.get("input#password").type(password);
+    cy.get('button[type="submit"]').click().wait(5000).then(() => {
+      cy.get("input#name").clear();
```

Comment:
Same semantic principles apply for all the inputs - these should be queried by label (e.g. `getByLabelText`). We should test from the *user's perspective*, not from the implementation side.

Reply:
After internal discussion, we've decided not to block approval on this one and instead forward this to the upcoming e2e test suite upgrade.